### DISCUSSION TITLE
Request Access (to game playtest) for bots

### DISCUSSION THREAD

**Discussion by pontifezo** [chr233/ASFEnhance#32] [功能建议 / Ideas]
Some games have the **Request Access** button on the Steam store page, that allows user to get instant (or later after the approval) access to the playtest.

E.g.:
https://store.steampowered.com/app/1591960/Randy_Blaster_3D/
https://store.steampowered.com/app/2164820/Orya/
https://store.steampowered.com/app/2175780/Dominance/

Is it possible to add the command for bots to Request Access ?

**Comment by chr233** [comment]
Working on it

**Comment by chr233** [comment]
https://github.com/chr233/ASFEnhance/releases/tag/1.6.15.0

Added, use `REQUESTACCESS` or `RA` command

**Reply by pontifezo** [reply]
Thank you!

**Comment by pontifezo** [comment]
By the way, what is the function of a TRANSFER command? Did not find the description in the Readme.

**Comment by chr233** [comment]
That's an command of origin ASF

**Reply by pontifezo** [reply]
Ah, got it, thanks.
